conda-forge.yml

```diff
@@ -4,3 +4,6 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

Comment:
@conda-forge/help-python is using CI services values other than `default` (as shown in the docs https://conda-forge.org/docs/maintainer/conda_forge_yml/#provider) strongly discouraged given the credits that `conda-forge` has? Or is experimentation with different CI services recommended to optimize build times overall?